### DISCUSSION TITLE
Helper for Sherlock's "Craft a blood rune" skill challenge should consider the True Blood Altar

### DISCUSSION THREAD

**Discussion by DustinKieler** [runelite/runelite#15023] [Ideas]
The skill challenge item requirement for the "Craft a blood rune" master clue step currently lists Dark essence fragments as a requirement.

If the player has completed Sins of the Father and has the level requirement for crafting blood runes, they can use pure essence at the true blood altar instead. Accessing the true blood altar through the Abyss would be faster to complete the step compared to the Kourend altar.

I confirmed that this does complete the skill challenge.

**Comment by Nightfirecat** [comment]
Presently, the plugin doesn't really have any system in place to handle alternative options for skill challenge clue completion. It's certainly something we could build out, but there are few cases where it applies--I guess this happens to be one of them.
